falcon/request.py

```diff
@@ -426,7 +426,12 @@ def __init__(self, env, options=None):
         # empty string, so normalize it in that case.
         path = env['PATH_INFO'] or '/'
 
-        if six.PY3:
+        if six.PY2:
```

Comment:
Could we flip this conditional statement to check for ``PY3`` first, since
* Python 3 is supposedly getting more popular than Python 2 (the latter would be end of life in 2020)
* Python 3 is less performant